php-practice_2.php

```diff
@@ -0,0 +1,115 @@
+<?php
+// Q1 tic-tac問題
+for ($i = 0; $i <= 100; $i++) {
+    if ($i % 4 === 0 && $i % 5 === 0) {
+        echo "tic-tac\n";
+    } else if ($i % 4 === 0) {
+        echo "tic\n";
+    } else if ($i % 5 === 0) {
+        echo "tac\n";
+    } else {
+        echo $i . "\n";
+    }
+}
+
+// Q2 多次元連想配列
+// 問題1
+$personalInfos = [
+    [
+        'name' => 'Aさん',
+        'mail' => 'aaa@mail.com',
+        'tel'  => '09011112222'
```

Comment:
[imo]
配列の最後にもカンマを追加した方が要素追加したときに追加した行しか差分がでないのでおすすめです。
```suggestion
        'mail' => 'aaa@mail.com',
        'tel'  => '09011112222',
```

Reply:
ありがとうございます。直します。